src/home_robot_hw/home_robot_hw/remote/api.py

```diff
@@ -183,7 +183,7 @@ def move_to_nav_posture(self):
 
         # First retract the robot's joints
         self.switch_to_manipulation_mode()
-        self.head.look_front(blocking=False)
+        self.head.look_front(blocking=True)
```

Comment:
Not really, I had to add significant sleeps in other places so head rotation can keep up with image-capture. Otherwise image-capture would fire long before camera has finished rotating.

Reply:
can we add a wait just like we do for images?